src/com/palmergames/bukkit/towny/command/TownyCommand.java

```diff
@@ -280,7 +288,15 @@ else if (split.length > 1 && split[1].equalsIgnoreCase("hud"))
 			} else if (split[0].equalsIgnoreCase("version") || split[0].equalsIgnoreCase("v")) {
 				if (!permSource.testPermission(player, PermissionNodes.TOWNY_COMMAND_TOWNY_VERSION.getNode(split[0].toLowerCase())))
 					throw new TownyException(Translation.of("msg_err_command_disable"));
-				TownyMessaging.sendMessage(player, towny_version);
+
+				if (TownyUpdateChecker.hasUpdate()) {

```

Comment:
```suggestion
				if (TownyUpdateChecker.shouldShowNotification()) {

```

I think both of these can be changed to the above and then they will also respect the new config setting.